CHANGELOG.md

```diff
@@ -3,6 +3,74 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.1.0
+
+_Mar 28, 2024_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Fix tooltip causing PieChart to crash on data change (#12571) @Rishi556
```

Comment:
Not sure this is worth a highlight, I don"t remember us highlighting any bug fix

Reply:
IMHO, we could highlight when it's a super major bug that a good portion of the community is waiting for.
But this should be an exception, we don't want to put to much light on the existence of bugs :laughing: